src/reconcile/rec_row.c

```diff
@@ -609,7 +609,8 @@ __rec_row_garbage_collect_fixup_update_list(WT_SESSION_IMPL *session, WT_RECONCI
     if (first_upd->type == WT_UPDATE_TOMBSTONE)
         return (0);
 
-    if (WT_TXNID_LT(first_upd->txnid, btree->oldest_live_txnid)) {
+    if (r->rec_last_checkpoint_timestamp != WT_TS_NONE &&
```

Comment:
Is this check OK? Does it mean that content can be wrtitten out in front of where it is legal to write it until the first checkpoint is created?

Reply:
The ingest table is in-memory. This just means we will not remove the key from memory until we bump to the first checkpoint. Everything will be kept in-memory in the ingest table if there is no checkpoint yet.